recipe/meta.yaml

```diff
@@ -13,6 +14,9 @@ source:
 
 build:
   number: 100
```

Comment:
It makes us cringe too, but unfortunately we had to get something ASAP so this is where @pelson and I are with it. Features with exclusions is what we really want to see.